src/arkode/arkode_erkstep.c

```diff
@@ -1061,6 +1061,29 @@ int erkStep_CheckButcherTable(ARKodeMem ark_mem)
     return(ARK_INVALID_TABLE);
   }
 
+  /* check if all b values are positive for relaxation */
+  if (ark_mem->relax_enabled)
+  {
+    if (step_mem->q < 2)
+    {
+      arkProcessError(ark_mem, ARK_INVALID_TABLE, "ARKODE::ERKStep",
+                      "erkStep_CheckButcherTables",
+                      "The Butcher table must be at least second order!");
+      return ARK_INVALID_TABLE;
+    }
+
+    for (i = 0; i < step_mem->stages; i++)
+    {
+      if (step_mem->B->b[i] < ZERO)
+      {
+        arkProcessError(ark_mem, ARK_INVALID_TABLE, "ARKODE::ERKStep",
+                        "erkStep_CheckButcherTable",
+                        "The Butcher table has a negative b value!");
+        return ARK_INVALID_TABLE;
+      }
+    }
+  }
+
```

Comment:
Again, it would be helpful if the documentation explicitly stated that use of relaxation along with an RK method with negative `b` coefficient will result in an error.

Reply:
Added a warning to the documentation about the method requirements